### ISSUE TITLE
Load a trained pbuf model into general pytorch env

### ISSUE THREAD

**Issue by r-kellerm** [intel/openfl#552]
**Is your feature request related to a problem? Please describe.**
I would like to test the trained model externally, using an independent pytorch-based script that compares several models. The pbuf format is serialized differently that the regular .pth torch snapshots and can't be loaded using model.load_state_dict.

**Describe the solution you'd like**
It would be great if you can provide either a tool for converting pbuf to pth, or a flag enabling to save the model in pth format during the learning rounds.

Thanks a lot.

**Comment by psfoley** [comment]
Thanks @r-kellerm for the suggestion! This is very timely. We are in the process of adding the ability to convert the model to native format after experiment completion. This will be part of our next OpenFL 1.5 release. Feel free to check out [that branch](https://github.com/psfoley/openfl/tree/save_native_model) and use `fx model save` after you run an experiment to get unblocked in the meantime (note: exact usage may change somewhat in the final version)

**Comment by r-kellerm** [comment]
Hello @psfoley , @igor-davidyuk ,

I'm using `fx model save -i model_best.pbuf -o model_best.pth` (with either best and last models), then load it using -
```
        ckpt = torch.load(args.model_path)
        model = CNN().to(device)
        model.load_state_dict(ckpt['model_state_dict'])
        model.eval()
```
However the model doesn't reach the same accuracy as the OpenFL training environment reports on. Am I missing a required step?
(The data  and the model are the same as in the federated environment)
Thanks!

**Comment by igor-davidyuk** [comment]
Hey, @r-kellerm, we probably should not discuss the resulting accuracy here, this issue is about functionality.
Still, it would be interesting to see metric logs and the test accuracy you measured to understand the order of values.
Please keep in mind that the best model snapshot is not always the last model snapshot, AND that Aggregator shows you averaged values for metrics during training.

**Comment by r-kellerm** [comment]
Hi @igor-davidyuk , @psfoley -
I reproduced the problem with `torch_cnn_mnist`.
The accuracy as reported from the training environment is >0.98.
When loading the saved model (with torch_cnn_mnist_best.pbuf converted to torch_cnn_mnist_best.pth) the reported accuracy is 0.58.
You can reproduce it with the attached file.
(The trained model was generated using `fx model save -i "./save/${TEMPLATE}_best.pbuf" -o $"./save/${TEMPLATE}_best.pth"`)
[standalone_mnist_cnn.py.txt](https://github.com/intel/openfl/files/10049742/standalone_mnist_cnn.py.txt)
